### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Update process to become a Maintainer based on PS-Committee discussion

cc @PowerShell/powershell-maintainers @PowerShell/powershell-committee

### CODE REVIEW

File: docs/maintainers/README.md
Comment (mirichmo) [inline docs/maintainers/README.md]: Can you make this an internal link? Here is how:
[Current Repository Maintainers](#current-repository-maintainers)

File: docs/maintainers/README.md
Comment (mirichmo) [inline docs/maintainers/README.md]: what about, "...individuals who **consistently** contribute in meaningful way..." instead?

**Review by mirichmo — CHANGES_REQUESTED** [review]

File: docs/maintainers/README.md
Comment (mirichmo) [inline docs/maintainers/README.md]: The updated wording of this makes the nomination process confusing. It implies that the Committee must approve a nomination without actually stating it. If the committee consultation is just advisory (does not require approval to add a new maintainer), then why is the committee consulted? Also, what constitutes approval? A majority or unanimous vote or maintainers or committee members? Should it be a unanimous vote of both maintainers and committee members? Majority of maintainers and unanimous committee? Unanimous maintainers and majority committee?

File: docs/maintainers/README.md
Comment (SteveL-MSFT) [inline docs/maintainers/README.md]: sounds good

File: docs/maintainers/README.md
Comment (SteveL-MSFT) [inline docs/maintainers/README.md]: Was trying to figure out the markdown to do that, but couldn't get it to work, can you put the raw markdown as code here?

File: docs/maintainers/README.md
Comment (SteveL-MSFT) [inline docs/maintainers/README.md]: My opinion is that the maintainers should be unanimous.  My thinking is that the Committee doesn't approve, but has veto power which I think a simple majority may be ok (I don't expect it to happen often/ever).  @joeyaiello @BrucePay thoughts?

File: docs/maintainers/README.md
Comment (mirichmo) [inline docs/maintainers/README.md]: I tried to, but it got rendered. It is a normal link, but set the link target to #current-repository-maintainers within the parenthesis.

Comment (SteveL-MSFT) [inline docs/maintainers/README.md]: Ok, the whitespace to dash is what I forgot.  This should work:

```markdown
[link](#current-repository-maintainers)
```

File: docs/maintainers/README.md
Comment (daxian-dbw) [inline docs/maintainers/README.md]: So that's "unanimous votes from maintainers to approve the nomination of a candidate, then majority votes from committee members to veto the nomination."
I think it's reasonable.

Comment (BrucePay) [inline docs/maintainers/README.md]: Agreed.

Comment (SteveL-MSFT) [inline docs/maintainers/README.md]: Updated

**Review by daxian-dbw — APPROVED** [review]
LGTM

**Review by mirichmo — APPROVED** [review]

**Comment by TravisEz13** [comment]
Make sure you use the markdownlint extension:  https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

**Comment by SteveL-MSFT** [comment]
@TravisEz13 unfortunately I've been editing this file in GitHub.  Fixing lint issues.

**Review by joeyaiello — APPROVED** [review]
LGTM, but I also think we should gate on approval from @BrucePay.

**Review by BrucePay — APPROVED** [review]

**Pull request merged**
